src/platform/Darwin/PlatformManagerImpl.h

```diff
@@ -58,8 +68,8 @@ class PlatformManagerImpl final : public PlatformManager, public Internal::Gener
 private:
     // ===== Methods that implement the PlatformManager abstract interface.
     CHIP_ERROR _InitChipStack();
-    void _Shutdown();
 
+#if CHIP_SYSTEM_CONFIG_USE_DISPATCH
```

Comment:
why was the shutdown removed? The PR description does not say anything about it.

If it needs to be removed because unused, why not a separate PR since it looks like very independent

Reply:
It was removed because it simply called the inherited method, which is the same as not overriding the method at all. I don't think this warrants a separate PR.